src/log-worker/createWorker.ts

```diff
@@ -0,0 +1,20 @@
+import Worker = require('tortoise');
+import { DotenvParseOutput } from 'dotenv';
+
+export const createWorker = async (
+  config: DotenvParseOutput
+): Promise<Worker> => {
+  const options: object = {
```

Comment:
It could be a wise thing to type the options object i.e create an interface that conforms to that.

Reply:
This has not been touched, my suggestion is that the object type is closely related to the any type. Some specific interface like QueueRetryPolicy or some better name should be used. It acts as documentation to start with.